### DISCUSSION TITLE
Support PowerShell?

### DISCUSSION THREAD

**Discussion by Liturgist** [github/codeql#13462] [General]
Is there any effort to support the PowerShell language?

**Comment by coadaflorin** [comment]
Hey @Liturgist, thanks for reaching out! 
At the moment support for PowerShell is not on our roadmap. You can check the [latest plans for CodeQL](https://github.com/orgs/github/projects/4247/views/1?filterQuery=label%3Acodeql) (and other GitHub offerings) on GitHub's public roadmap. This is where we'll announce support for new languages.
